stock_release_channel_shipment_advice_process_end_time/views/stock_release_channel.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 ACSONE SA/NV
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record model="ir.ui.view" id="stock_release_channel_form_view">
+        <field name="model">stock.release.channel</field>
+        <field
+            name="inherit_id"
+            ref="stock_release_channel_process_end_time.stock_release_channel_form_view"
+        />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='process_end_time']" position="after">
+                <field name="process_end_time_delay" />
```

Comment:
@sbejaoui It would be nice to display the word 'minutes' next to the input field ...
```suggestion
                <field name="process_end_time_delay" />
                <label for="process_end_time_delay" />
                    <div>
                        <field name="process_end_time_delay" class="oe_inline" />
                        <label string="minutes"  class="oe_inline" style="margin-left:5px" />
                   </div>
```